### ISSUE TITLE
MUD School and Newbie Outfit Set (except light) Needs Updating

### ISSUE THREAD

**Issue by Qethia** [prophecymud/ideas#22]
I think the current MUD school environment is infantile and potentially off-putting to new players. The initial combat training is especially cringeworthy (ex. "That clucking hurt!" or "A bunny rabbit screams, 'But it's DUCK season! Not rabbit season!!"). It looks like a child's game, or worse, and is therefore misleading. Also, the eq drops/outfit pieces are worse than useless once they hit Sendar (worse because they all have to be replaced except the light). I'd like to see viable, non-melt_drop items provided automatically to new players, including weapon. Not that gathering newbie kits is any kind of problem, but at this point we can't guarantee someone will always be around to provide them as soon as they leave the MUD school). Ideally, new players could gather all of the current newbie equipment and weapons themselves, but this is not always possible (ex. the greaves from Riva, the Ulgo ruby gem). Stocking equivalent eq/weapons in areas that are within level <10 newbie walking distance (very few) or even the school's practice yard is an alternative option, although simply loading them up on creation may be the easier solution. Lower Nyissan Jungle is the only <10 newbie area that loads decent eq/weapons, but you need to be wearing at least a few "good" pieces to begin with in order to be able to kill the mobs to get them. In a perfect "world", I'd like to see eq/weapon drops in areas at the level of those who would be able to use them (meaning not having to be level 25 to get level 1 equipment). Assuming that the stats/leveling issue is resolved soon, even a few viable pieces of equipment and a decent weapon replacing the current "outfit" melt_drop items and giving new players a fighting chance at surviving initial combat encounters immediately would be a major QoL improvement for what is arguably our most important resource.

**Comment by prophnoran** [comment]
Ideally imo it'd be possible to level up from 1 without any help at all; stuff for use at 25 should drop from mobs you can kill at 15 or whatever

**Comment by anhegofcherek** [comment]
> I think the current MUD school environment is infantile and potentially off-putting to new players

I can see that, and would support someone attempting to rebuild it

> Also, the eq drops/outfit pieces are worse than useless once they hit Sendar (worse because they all have to be replaced except the light)

Removing stats effect on gains (#4) should help here - it won't make them useful, but at least they won't be "harmful"!

> I'd like to see viable, non-melt_drop items provided automatically to new players, including weapon [...] Ideally, new players could gather all of the current newbie equipment and weapons themselves

I'd like new characters to not need external assistance to play and enjoy and succeed, definitely.

I don't want the solution to be "automatically starting off with a set of optimised newbie equipment" because that cuts out avenues of discovery and exploration and variation.

With stats not affecting gains (#4) then they can use "anything that works well" and there's a lot more breadth - suddenly anything they loot or buy becomes potentially viable, whereas right now  things are much more constrained.

> In a perfect "world", I'd like to see eq/weapon drops in areas at the level of those who would be able to use them (meaning not having to be level 25 to get level 1 equipment).

I think I agree with the intention but not necessarily with the proposal.

I'm ok with needing to be level 25 to get the _best_ level 1 shield. I just think there should be a lot more shields etc.

I definitely agree that people should be able to equip themselves adequately and well (and again, I think #4 should help there!)

**Comment by grekan** [comment]
I agree mud school while it looked cute at the time, perhaps isn't the best intro environment anymore. 

I think it would be a good idea to make something that is 75% as good a "fully best newbie set" and they can go off to Arctic or Lake Sendar on their own.
